packages/core/src/components/overlay/overlay.tsx

```diff
@@ -215,18 +215,11 @@ export class Overlay extends AbstractPureComponent<OverlayProps, OverlayState> {
         }
 
         // decorate the child with a few injected props
-        const tabIndex = this.props.enforceFocus || this.props.autoFocus ? 0 : undefined;
```

Comment:
This `tabIndex={0}` doesn't do anything because it's not on an interactive element. And `autoFocus` brings focus to `startFocusTrapElement`, not this element.